rmw_connextdds_common/src/common/rmw_subscription.cpp

```diff
@@ -98,6 +98,14 @@ rmw_api_connextdds_create_subscription(
     }
   }
 
+  // Adapt any 'best available' QoS options
+  rmw_qos_profile_t adapted_qos_policies = *qos_policies;
+  rmw_ret_t ret = rmw_dds_common::qos_profile_get_best_available_for_topic_subscription(
+    node, topic_name, &adapted_qos_policies, rmw_get_publishers_info_by_topic);
```

Comment:
I think this should fix the Windows build error: https://ci.ros2.org/job/ci_windows/17027
```suggestion
    node, topic_name, &adapted_qos_policies, rmw_api_connextdds_get_publishers_info_by_topic);
```

Reply:
thanks! works fine in my local Linux system

Reply:
another run of CI 
* Linux [![Build Status](http://ci.ros2.org/buildStatus/icon?job=ci_linux&build=16631)](http://ci.ros2.org/job/ci_linux/16631/)
* Linux-aarch64 [![Build Status](http://ci.ros2.org/buildStatus/icon?job=ci_linux-aarch64&build=11213)](http://ci.ros2.org/job/ci_linux-aarch64/11213/)
* Windows [![Build Status](http://ci.ros2.org/buildStatus/icon?job=ci_windows&build=17028)](http://ci.ros2.org/job/ci_windows/17028/)

Reply:
CI is looking at my branch; I just synced my branch with yours and re-triggered CI: https://github.com/ros2/rmw/pull/320#issue-1205954246